src/create/CreatePage.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function Create() {
+
+  return (  
+  <Hello>Hallo</Hello>
+    )}
```

Comment:
Did you forget to run your formatter? The indentation is a little off...

```suggestion
  return (  
    <Hello>Hallo</Hello>
  )}
```